editoast/src/models/rolling_stock/mod.rs

```diff
@@ -251,35 +251,35 @@ pub mod tests {
     use super::RollingStockModel;
     use actix_web::web::Data;
 
-    pub fn get_fast_rolling_stock() -> RollingStockModel {
```

Comment:
You could move this function in fixtures.rs and call it from `named_fast_rolling_stock`